main.js

```diff
@@ -0,0 +1,75 @@
+var button = document.querySelector('button');
+var input = document.querySelector('input');
+var h1 = document.querySelector('h1');
+
+
+// button adds event and starts timer
+
+var addText = function (){
+  // create <p>
+  var p = document.createElement('p');
+  // store value from input
+  var storeInput = input.value;
+  p.textContent = "Currently working on..." + storeInput;
+
+var body = document.querySelector('body');
+  body.appendChild(p);
+
+};
+
+button.addEventListener('click', addText);
+
+
+// add counter
+
+var number = 25;
+var intervalID;
+
+
+var countUp = function(){
```

Comment:
This function is named "countUp" yet it's counting down... There's something in coding called "The principle of lease surprise" Would this adhere to it?